Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -424,6 +425,12 @@ function traverseSceneGraph(sceneGraph, node, transformToRoot) {
     sceneGraph._skinnedNodes.push(index);
   }
 
+  // Create and store the public version of the runtime node.
+  const model = sceneGraph._model;
+  const publicNode = new ModelExperimentalNode(model, runtimeNode);
+  const name = publicNode.name;
+  model._nodesByName[name] = publicNode;
```

Comment:
what should be the behavior if `name` is undefined? (name is not a required field). Do we skip creating a public node since there's no way to reference it?

Reply:
`Model` didn't account for this, but it makes sense to ignore the node if we can't reference it by name.